CruCentralCoast/Community/Missions/MissionDetailsVC.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import UIKit
+import SafariServices
```

Comment:
It looks like you don't use this anymore, in which case it should be removed

Reply:
removed!